src/main/java/net/snowflake/ingest/connection/ServiceResponseHandler.java

```diff
@@ -60,20 +60,8 @@ public static IngestResponse unmarshallIngestResponse(HttpResponse response)
       throw new IllegalArgumentException();
     }
 
-    // Grab the status line from the response
-    StatusLine statusLine = response.getStatusLine();
-
-    // If we didn't get a good status code, handle it
-    if (!isStatusOK(statusLine)) {
-
-      // Exception status
-      LOGGER.warn(
```

Comment:
So this warn error message is kind of useful to find out which API has non ok status.. 

Can you create an Enum with API names and pass it down to handleExceptionalStatus and print the enum along with logger.warn in switch case??

Reply:
It will be hard to debug right now on which API has failed from Client SDK side. and we will be calling multiple APIs from KC. It would be better to distinguish between APIs and failures. Lets also print requestId in warn if possible. 

Unless this error is surfaced upwards and its being logged at top level.